includes/class-wpcom-legacy-redirector.php

```diff
@@ -68,13 +78,22 @@ static function register_redirect_custom_capability() {
 	}
 
 	/**
-	 * Remove Bulk Edit from the Bulk Actions drop-down on the CPT's edit screen
+	 * Remove Bulk Edit from the Bulk Actions drop-down on the CPT's edit screen UI.
+	 *
+	 * @param array $actions current bulk actions available to drop-down.
+	 *
+	 * @return array $actions available bulk actions minus edit functionality.
 	 */
 	static function remove_bulk_edit( $actions ) {
 		unset( $actions['edit'] );
 		return $actions;
 	}
 
+	/**
+	 * Performs redirect if current URL is a 404 and redirect rule exists.
+	 *
+	 * @return void redirect to destination URL if matching rule found.
```

Comment:
`@return void` should not be used - see https://developer.wordpress.org/coding-standards/inline-documentation-standards/php/#1-functions-class-methods